ldapAdapterEXT-ext-impl/src/main/java/com/bosch/osmi/sw360/ldapAdapter/CustomPortalLDAPImporterImpl.java

```diff
@@ -131,5 +164,80 @@ private Organization addOrganization(String organizationName, User user) throws
 		}
         return organization;
 	}
+
+	private String mapOrganizationName(String name) {
+		loadLDAPImporterProperties();
```

Comment:
Yeah, I actually had in loaded statically, but then changed to let it reload mappings on the fly. I agree: I'll make it static again.